modules/home/assets/js/components/HomeScreen.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+import { Box } from '@elementor/ui';
+import TopSection from './TopSection';
+
+const HomeScreen = () => {
+	const [topScreenProps, setTopScreenProps] = useState({videoUrl: 'https://elementor.com/academy/getting-started-with-elementor/', ctaUrl: 'http://wordpress-dev.local/wp-admin/edit.php?elementor_new_post&post_type=page', embedUrl: 'https://www.youtube.com/embed/icTcREd1tAg?si=MPamCEWNeRR_VdAn&amp;controls=0'});
```

Comment:
```suggestion
	const [ topScreenProps, setTopScreenProps ] = useState( { videoUrl: 'https://elementor.com/academy/getting-started-with-elementor/', ctaUrl: 'http://wordpress-dev.local/wp-admin/edit.php?elementor_new_post&post_type=page', embedUrl: 'https://www.youtube.com/embed/icTcREd1tAg?si=MPamCEWNeRR_VdAn&amp;controls=0' } );
```

Reply:
Compare `modules/display-conditions/assets/js/editor/components/content.js` in the pro repo.

Reply:
Can we extract the url variables already, so that we can update it easier?